sql/ddl/alter/add-trigger-2.md

```diff
@@ -0,0 +1,135 @@
+---
+author: kapnobatai136
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+inAlgoPool: false
+
+standards:
+  sql.define-tables.2: 10
+
+tags:
+
+  - workout
+
+  - deep
+
+links:
+
+  - '[More on CREATE TRIGGER](https://www.codeproject.com/Articles/25600/Triggers-SQL-Server){website}'
+
+aspects:
+  - workout
+  - deep
+
+
+---
+
+# Add Trigger (2)
+
+---
+## Content
+
+In the previous insight we showed you two arguments, `RESTRICT` and `NO ACTION`, which are used to restrict deleting records that have a matching row in the child tables. It's also possible to create triggers that modify the data in a table using arguments such as `CASCADE`, `SET NULL`, and `SET DEFAULT`.
+
+### CASCADE
+
+Using the `CASCADE` argument means that when a row in the parent table is deleted or updated, all the matching rows in the child tables are deleted or updated. Here is an example syntax:
```

Comment:
Maybe add a "too" at the end of the first sentence?

Using X says that when parent is updated, the child is updated too.